snmp/datadog_checks/snmp/data/default_profiles/riverbed-steelhead.yaml

```diff
@@ -0,0 +1,53 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "riverbed"
```

Comment:
since you have [another riverbed profile](https://github.com/DataDog/integrations-core/pull/15294#), you can just create a `riverbed.yaml` profile with this part and the sysoid `1.3.6.1.4.1.17163.1.*` to cover more riverbed products